manifests/config.pp

```diff
@@ -66,6 +66,7 @@
   $proxy_buffers                  = '32 4k',
   $proxy_buffer_size              = '8k',
   $proxy_cache_inactive           = '20m',
+  $proxy_cache_key                = '$scheme$proxy_host$request_uri',
```

Comment:
Yep, but the nginx::config::proxy_cache_key parameter you created here isn't referenced anywhere else.